tests/test_logger.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any
+from unittest import TestCase
+import requests
+from dor.providers.package_generator import PackageResult
+from utils.logger import LogStatus, Logger
+
+
+class TestLoggerIntegration(TestCase):
+
+    def setUp(self):
+        self.collection_name = "test_logs"
+        self.logger = Logger(collection_name=self.collection_name)
+        self.pocket_base_url = "http://pocketbase:8080"
+        self.pocket_base_username = "test@umich.edu"
+        self.pocket_base_password = "testumich"
```

Comment:
I don't think these are being used, right? Since `Logger` is depending on `config` currently.